src/Components/Utils/SnackBar/index.js

```diff
@@ -0,0 +1,54 @@
+/* @flow */
+import React from 'react';
+import MatieralSnackBar from 'material-ui/Snackbar';
```

Comment:
MaterialSnackBar?

Reply:
meant that its misspellen -> MatieralSnackBar

Reply:
🙈
https://github.com/veritus/veritus-web/pull/54/commits/6d21a373152db40eac531d72bca814cd210d317e#diff-ec2ba75bb022595f14afa25ea72fc3a4R3